router/src/main.rs

```diff
@@ -401,6 +420,41 @@ pub async fn get_base_tokenizer(api: &Api, api_repo: &ApiRepo) -> Option<Tokeniz
     }
 }
 
+/// get tokenizer_config from the Huggingface Hub
```

Comment:
nit: since the last commit we added hf_hub as a dependency and we could use api_repo.get here directly.